### PR TITLE
feat: initialise OSM adapter for Meshery

### PR DESCRIPTION
**Pull request by kushthedude**
Signed-off-by: Kush Trivedi <kushthedude@gmail.com>

### CODE REVIEW

File: ui/components/MesheryPerformanceComponent.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ const meshes = [
   'Octarine',
   'Rotor',
   'SOFAMesh',
+  'osm',
```
Comment (leecalcote) [inline ui/components/MesheryPerformanceComponent.js:40]: Capitalization here would be good.

File: ui/components/Navigator.js [hunk 2]
```diff
@@ -412,7 +414,7 @@ class Navigator extends React.Component {
     }
 
     handleAdapterClick = (id, link) => {
-      let allowedId = ["Consul", "Istio", "Linkerd", "Network Service Mesh", "Octarine", "Citrix Service Mesh"];
+      let allowedId = ["Consul", "Istio", "Linkerd", "Network Service Mesh", "Octarine", "Citrix Service Mesh", "osm"];
```
Comment (leecalcote) [inline ui/components/Navigator.js:417]: Capitalize, please.

File: ui/public/static/img/osm-white.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...ath d="m203.895408 790.87234 1354.028262-794.36565" mask="url(#b)"/><path d="m554.724703 737.356007 1354.028267-794.36565" mask="url(#b)"/><path d="m548.778444 903.851266 1354.028266-794.36565" mask="url(#b)"/><g mask="url(#b)"><g transform="matrix(-1 0 0 1 1917.74975 -401.91278)"><path d="m6.546271 794.385751 1354.028269-794.36565"/><path d="m.600012 972.773528 1354.028268-794.36565"/><path d="m351.429307 919.257195 1354.028263-794.36565"/><path d="m345.483048 1085.75245 1354.028262-794.365646"/><path d="m517.924566 1145.21505 1354.028264-794.365654"/><path d="m868.753861 1091.69871 1354.028269-794.365647"/><path d="m862.807602 1258.19397 1354.028268-794.365648"/></g></g></g></g><g transform="translate(129.637675 68.695502)"><path d="m.83391004 416.790392v-354.4644467l264.99999996 506.9298167z" fill="#1128a9"/><path d="m325.83391 438.925408 220.024719-126.457032v-311.81207645l-220.024719 122.94614145z" fill="#569cf5"/><path d="m0 245.023614 178.007865 104.748898v-243.433665l-178.007865-106.338847z" fill="#fb8143" transform="translate(414.362325 140.925408)"/><path d="m325.83391 438.925408 220.024719-126.457032v-311.81207645l-220.024719 122.94614145z" fill="#569cf5"/></g></g></svg>
```
Comment (leecalcote) [inline ui/public/static/img/osm-white.svg:1]: This SVG needs converted to grayscale.

File: ui/components/MesheryPerformanceComponent.js (hunk 1 above)
Comment (kushthedude) [reply ui/components/MesheryPerformanceComponent.js]: done

File: ui/components/Navigator.js (hunk 2 above)
Comment (kushthedude) [reply ui/components/Navigator.js]: done

File: ui/public/static/img/osm-white.svg (hunk 3 above)
Comment (kushthedude) [reply ui/public/static/img/osm-white.svg]: added png, not good in editing :'(

**Comment by leecalcote** [comment]
Small tweaks added.

**Comment by kushthedude** [comment]
> Small tweaks added.

Great!

**Pull request merged**